### PR TITLE
dotting the i's

### PR DESCRIPTION
**Pull request by alinagator**

### CODE REVIEW

File: posts/goodbye-whatsapp.md [hunk 1]
```diff
@@ -69,10 +69,10 @@ This is the same for Whatsapp. The chunky content you receive like files, videos
...
+The huge swaths of data allow for incredible levels of targeting. The more privacy and data you give up, the more targetable and manipulable you are. 'But *I'm* not influenced by ads' - sadly you may not be aware of it. 93% of Americans believe they are in the top 50% of drivers - this is [illusory superiority](https://en.wikipedia.org/wiki/Illusory_superiority). And based on the analytics, you and I very much are influenceable. Even if you are unmovable, frequently it's our friends unintentionally giving up our data; uploading photos of us, tagging us, and merely being a part of your network allows data scientists to make highly accurate guesses about who you are - the inverse is true.
```
Comment (alinagator) [inline posts/goodbye-whatsapp.md:75]: > the inverse is true

...the inverse of what? Is it referring to "Even if you unmovable"?

File: posts/goodbye-whatsapp.md [hunk 2]
```diff
@@ -36,15 +36,15 @@ One way to do that is by not using it's platform or the platforms like Whatsapp
...
+Despite what Zuckerberg might say publicly, it is deeply embedded into Facebook's DNA to get your data and sell it - it is not there to offer you privacy or the best private messaging service; that's an afterthought. As with this update to the t&cs they are demonstrating that they can change the rules whenever they like to what they most prefer - more of your data. [A tweet](https://twitter.com/Shiftreduce/status/1347546599384346624) revealed a subtle change of Whatsapp's end-to-end encryption paper, which is a little concerning:
 
 ![](/content/images/2021/whatsapp-policy-diff.png)
 
 ## 💰 Whatsapp founder gave $50 mil to Signal
-Yes that's right, the co-founder of Whatsapp [gave $50 million dollars to Signal](https://signal.org/blog/signal-foundation/) to it's competitor. Why? He regrets selling to Facebook, when the [Facebook Cambridge Analytica scandal](https://www.bbc.com/news/technology-54722362) happened several years back, he tweeted ['It is time. #deletefacebook'](https://twitter.com/brianacton/status/976231995846963201). Both Founders ended up leaving over disputes of monetisation - well we are here now and it's time to delete Whatsapp too.
```
Comment (alinagator) [inline posts/goodbye-whatsapp.md:44]: > well we are here now and...

Wasn't sure what this meant (where's 'here'?), and felt it kind of diluted the punchiness. Also not sure "Both Founders ended up leaving over disputes of monetisation" is relevant to this point.

File: posts/goodbye-whatsapp.md [hunk 3]
```diff
@@ -18,13 +18,13 @@ Anyone who uses Whatsapp would have seen the ultimatum Facebook is handing its u
 
 In 2014 [Facebook bought Whatsapp for $19 billion](https://www.bbc.com/news/business-26266689). This was not out of charity. The intention was always to [integrate the valuable data](https://mashable.com/article/facebook-data-merge-privacy-concerns/) on you and monetise this huge purchase.
 
-Facebook's primary business is collecting and storing as much data about you to build complex models most people don't understand. But that doesn't matter as long as they can keep your eyeballs on their products - they can sell your attention to whoever wants to buy it, from consumer product companies to people trying to manipulate election outcomes.
```
Comment (alinagator) [inline posts/goodbye-whatsapp.md:21]: > But that doesn't matter as long as...

Which part of the previous sentence is that meant to refer to?

File: posts/goodbye-whatsapp.md (hunk 1 above)
Comment (ucg8j) [reply posts/goodbye-whatsapp.md:75]: Inverse, of your friends sharing info on you, you can remove the privacy of your friends by sharing. I agree this is not clear and the point is made without the addition.

File: posts/goodbye-whatsapp.md (hunk 3 above)
Comment (ucg8j) [reply posts/goodbye-whatsapp.md:21]: Good pt, I was trying to make the point that even though the models aren't understood (in the causal sense), understanding the model doesn't matter as long as the objective function is acheived.

Comment (ucg8j) [reply posts/goodbye-whatsapp.md:21]: Tried to clarify

**Pull request merged**